invenio_rdm_records/theme/views.py

```diff
@@ -160,3 +124,11 @@ def vocabulary_subtitle(dict_key, vocabulary_key):
     vocabulary = Vocabularies.get_vocabulary(vocabulary_key)
     return vocabulary.get_title_by_dict(dict_key).split('/')[1].strip() \
         if vocabulary else ""
+
+
+@blueprint.app_template_filter('serialize_ui')
+def serialize_ui(record):
+    """Returns the UI serialization of a record."""
+    serializer = UIJSONSerializer()
+    # We need a dict not a string
+    return json.loads(serializer.serialize_object(record))
```

Comment:
Minor: Could we instead refactor UIJSONSerializer a bit so that we can get the dict. Something a long the lines:

```
def serialize_object(self, obj, response_ctx=None, *args, **kwargs):
        return json.dumps(self.get_data(obj))

def get_data(self, obj):
      self._serialize_obj_ui(obj)
      self._serialize_obj_dates(obj)
      return obj
```

Then you can this instead:

```return UIJSONSerializer().get_data(record)```

Reply:
done :)